testsuite/features/build_validation/init_clients/proxy.feature

```diff
@@ -12,6 +12,9 @@ Feature: Setup SUSE Manager proxy
 
   Scenario: Clean up sumaform leftovers on a SUSE Manager proxy
     When I perform a full salt minion cleanup on "proxy"
+
+  Scenario: Install proxy pattern on the proxy
```

Comment:
This title say less that it has

Reply:
Hmmm, unsure what you mean

Anyway... I have to change that part, for other reasons.

Reply:
Ah, ok. Right.